src/js/configuration_panel/configuration-panel.js

```diff
@@ -86,16 +86,16 @@ https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
                 openNewWindow: "Open a new browser window",
                 goToPreviousWindow: "Switch to the previous browser window",
                 goToNextWindow: "Switch to the next browser window",
-                zoomIn: "Zoom-in on the current tab",
-                zoomOut: "Zoom-out on the current tab"
+                zoomIn: "Zoom-in on the active webpage",
```

Comment:
Sorry to nitpick but "web page" is probably better.

Reply:
Whoops! My bad. I'll fix it now.

Reply:
@the-t-in-rtf it has been changed. Thanks for correcting me here.